Os/CMakeLists.txt

```diff
@@ -169,7 +169,6 @@ endif()
 # If baremetal scheduler is set, remove the previouse task files and add in the Baremetal variant
 if (BAREMETAL_SCHEDULER)
```

Comment:
Here the build system must compile extra code for the scheduler. This is added to the Os package, so it must trigger off a CMake variable, not just a #define variable.